server/services/offendersService.js

```diff
@@ -83,7 +83,7 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
 
       const dbManualInProgress = await formService.getCategorisationRecords(
         agencyId,
-        [Status.STARTED.name, Status.SECURITY_BACK.name, Status.SUPERVISOR_BACK.name],
+        [Status.STARTED.name, Status.SECURITY_BACK.name, Status.SUPERVISOR_BACK.name, Status.SECURITY_AUTO],
         CatType.INITIAL.name,
```

Comment:
As discussed @steverendell - we would want to include any in security auto state for both initial and recats

Reply:
Sorry .. I think we need manual and flagged as well as auto 🤔